portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,27 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+var slideIndices = [1, 1, 1];
+var slideSections = ["ed-slides", "project-slides", "work-slides"];
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+for (i = 0; i < 3; i++) {
+    showSlides(slideIndices[i], i);
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function plusSlides(n, o) {
+    showSlides(slideIndices[o] += n, o);
```

Comment:
I would pull the += out and do something like:

slideIndices[o] += n;
showSlides(slideIndices[o], o);

to make it more clear what the order of operations is and that the updated value is what's passed into the function call.

Reply:
I'll add more clarity